modern/src/MainPage.js

```diff
@@ -158,7 +158,7 @@ const MainPage = () => {
           </Toolbar>
         </Paper>
         <div className={classes.deviceList}>
-          <DevicesList />
+          <DevicesList filter={deviceName} />
```

Comment:
Can we rename `deviceName` variable to something better? Like `searchKeyword` or something.

Reply:
Sure.